var/spack/repos/builtin/packages/revbayes/package.py

```diff
@@ -40,6 +41,6 @@ def regenerate(self):
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
         if '+mpi' in spec:
-            install('rb-mpi', prefix.bin)
+            install(join_path(self.build_directory, '..', 'rb-mpi'), prefix.bin)
```

Comment:
Does this not work anymore?

Reply:
https://github.com/revbayes/revbayes/blob/master/projects/cmake/regenerate.sh#L229

The build script puts the executable one directory up from the directory you run it in. Without this change my builds fail on this step complaining they can't find the executable.

Reply:
Agree - if I attempt to reinstall old versions that previously worked, the binaries are up one directory from where they should be.  Not sure what caused this behavior change since I'm pulling down github releases that previously worked... But this change seems fine